### PR TITLE
Add Kimai config example

### PR DESCRIPTION
**Pull request by Elbullazul**

### CODE REVIEW

File: example_configs/kimai.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# See https://www.kimai.org/documentation/local-yaml.html
+# this file should be renamed local.yaml
+kimai:
+  ldap:
+    activate: true
+    connection:
+      host: lldap
+      port: 3890
+      useSsl: false
```
Comment (nitnelave) [inline example_configs/kimai.yaml:9]: Maybe add a comment "true if using ldaps"?

File: example_configs/kimai.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      username: uid=admin,ou=people,dc=example,dc=com
+      password: <PASSWORD_HERE>
+      accountFilterFormat: (&(objectClass=person)(uid=%s))
+      bindRequiresDn: true
+      optReferrals: false
+
+    user:
+      baseDn: ou=people, dc=example, dc=com
+      usernameAttribute: uid
+      filter: (&(objectClass=person)(memberof=cn=kimai,ou=groups,dc=example,dc=com))
```
Comment (nitnelave) [inline example_configs/kimai.yaml:20]: Add a top level note that you need to create the "kimai" group and add users to it

**Review by nitnelave — CHANGES_REQUESTED** [review]
Looks good, thanks !

File: example_configs/kimai.yaml (hunk 2 above)
Comment (Elbullazul) [reply example_configs/kimai.yaml:20]: Would moving the group filtering condition to a comment do the trick? Kimai doesn't require a group for LDAP to work.

Comment (nitnelave) [reply example_configs/kimai.yaml:20]: Yeah, you can move it to a comment, but still add a sentence (in the same comment) about the need to create the group.

**Review by nitnelave — APPROVED** [review]

**Pull request merged**